#!/usr/bin/env python
PACKAGE = "irols"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('xy_vel_p', double_t, 0, 'XY Velocity Proportional Gain', 1.5, 0, 20)
gen.add('xy_vel_i', double_t, 0, 'XY Velocity Integral Gain', 0.012, 0, 1.2)
gen.add('xy_vel_d', double_t, 0, 'XY Velocity Derivative Gain', 0.2, 0, 5)

exit(gen.generate(PACKAGE, "controller", "pid_gains"))
